src/quacc/calculators/espresso/opt_patchs.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+
+from quacc.recipes.espresso.core import post_processing_job
+from quacc.utils.dicts import recursive_dict_merge
+
+
+class PostProcessingPatch:
+    job_names = ["post_processing"]
+
+    def __init__(self, **calc_kwargs):
+        self._results = {job_name: {} for job_name in self.job_names}
+        self._calc_kwargs = calc_kwargs
+
+    def step(self, *args, **kwargs):
+        super().step(*args, **kwargs)
+        # Would this cause issues because everything is ran asynchrously?
+        # If the opt is fast compared to the post-processing, file could be
+        # overwritten while being copied or something like that?
```

Comment:
If my understanding is correct here post_processing will be able to ran concurrently. If the simulation is fast enough, files could changed while being copied?

Reply:
Indeed that seems like it could be problematic if you are launching separate post processing jobs in parallel to the running optimization. Generally, dealing with file I/O and concurrent simulations is tricky (hence all the custom features you had to add in when making your Espresso recipes).